agent/ae/ae.go

```diff
@@ -13,32 +13,6 @@ import (
 	"github.com/hashicorp/consul/logging"
 )
 
-// scaleThreshold is the number of nodes after which regular sync runs are
-// spread out farther apart. The value should be a power of 2 since the
-// scale function uses log2.
-//
-// When set to 128 nodes the delay between regular runs is doubled when the
-// cluster is larger than 128 nodes. It doubles again when it passes 256
-// nodes, and again at 512 nodes and so forth. At 8192 nodes, the delay
-// factor is 8.
-//
-// If you update this, you may need to adjust the tuning of
-// CoordinateUpdatePeriod and CoordinateUpdateMaxBatchSize.
-const scaleThreshold = 128
-
-// scaleFactor returns a factor by which the next sync run should be delayed to
-// avoid saturation of the cluster. The larger the cluster grows the farther
-// the sync runs should be spread apart.
-//
-// The current implementation uses a log2 scale which doubles the delay between
-// runs every time the cluster doubles in size.
-func scaleFactor(nodes int) int {
-	if nodes <= scaleThreshold {
-		return 1.0
-	}
-	return int(math.Ceil(math.Log2(float64(nodes))-math.Log2(float64(scaleThreshold))) + 1.0)
-}
```

Comment:
This was moved to lower down in the file, next to where it is used. This is a minor part of the implementation, so it really should not be the first thing in the file.